### ISSUE TITLE
NTR: Creatine low in CSF (Lab: cerebrospinal fluid tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8450]
New term request Creatine low in CSF (segal_090513170927): Cerebrospinal fluid

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000400
- label: Reduced CSF creatine concentration
- parents: [Abnormal CSF metabolite concentration (HP:0025454)](https://hpo.jax.org/app/browse/term/HP:0025454)
- pmid: 
- def: The concentration of creatine in the cerebrospinal fluid (CSF) is below the lower limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8450
- synonyms:
